docs/INSTALL.md

```diff
@@ -68,6 +68,23 @@ cd ~/5gdeploy
 # change 'ueransim' to the image that you want to rebuild
 ```
 
+### Installation Options
+The following optional arguments can be passed to `./install.sh`:
+* `--bridge-version <version>`: Specify a **branch**, **tag**, or **commit hash** to use for bridge (_default: [9ba97f1735022fb5f811d9c2a304dda33fae1ad1](https://github.com/jpetazzo/pipework)_)
```

Comment:
This should be named `--pipework-version`.
*bridge* is a container built by 5gdeploy, which does not have a version.
Only `pipework` has a version.